### ISSUE TITLE
CASDecom not working if drop_occ != cas_frozen

### ISSUE THREAD

**Issue by gustavojra** [FermiQC/Fermi.jl#25]
For C2 TZ

```
   • CAS Decomposition started:
Active Occupied Orbitals: [3, 4, 5, 6]
Active Virtual Orbitals:  [7, 8, 9, 10]
Getting T1 and T2...Done.
Computing ecT1, ecT2
3 out of 6
ERROR: BoundsError: attempt to access 4×4×4×4 Array{Float64,4} at index [6, 6, 2, 2]
Stacktrace:
 [1] setindex!(::Array{Float64,4}, ::Float64, ::Int64, ::Int64, ::Int64, ::Vararg{Int64,N} where N) at ./array.jl:828
 [2] get_casT4αβ!(::Array{Float64,4}, ::Int64, ::Int64, ::Int64, ::Int64, ::Array{Float64,1}, ::Array{Fermi.ConfigurationInteraction.Determinant,1}, ::Array{Float64,1}, ::Array{Fermi.ConfigurationInteraction.Determinant,1}, ::Fermi.ConfigurationInteraction.Determinant, ::Int64, ::Int64, ::Array{Float64,2}, ::Array{Float64,4}, ::Array{Float64,4}, ::Array{Float64,4}) at /home/mmd01986/.julia/dev/Fermi.jl/src/Methods/CoupledCluster/ecRCCSD/CTF.jl:509
 [3] cas_decomposition(::Tuple{Fermi.ConfigurationInteraction.Determinant,Array{Float64,1},Array{Fermi.ConfigurationInteraction.Determinant,1},Array{Float64,1},Array{Fermi.ConfigurationInteraction.Determinant,1},Array{Float64,1},Array{Fermi.ConfigurationInteraction.Determinant,1}}, ::Int64, ::Int64, ::Array{Int64,1}, ::Array{Int64,1}, ::Array{Float64,2}, ::Array{Float64,4}, ::Array{Float64,4}, ::Array{Float64,4}) at /home/mmd01986/.julia/dev/Fermi.jl/src/Methods/CoupledCluster/ecRCCSD/CTF.jl:925
 [4] macro expansion at ./util.jl:234 [inlined]
 [5] Fermi.CoupledCluster.ecRCCSD{Float64}(::Fermi.ConfigurationInteraction.CASCI{Float64}, ::Fermi.CoupledCluster.CTF) at /home/mmd01986/.julia/dev/Fermi.jl/src/Methods/CoupledCluster/ecRCCSD/CTF.jl:51
 [6] top-level scope at REPL[117]:1
 [7] eval(::Module, ::Any) at ./boot.jl:331
 [8] eval_user_input(::Any, ::REPL.REPLBackend) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.4/REPL/src/REPL.jl:86
 [9] run_backend(::REPL.REPLBackend) at /home/mmd01986/.julia/packages/Revise/BqeJF/src/Revise.jl:1184
 [10] top-level scope at REPL[1]:0
```

**Comment by mdav2** [comment]
@gustavojra this was fixed in #39 right?

**Comment by gustavojra** [comment]
Fixed #39
